shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/insert/values/InsertValueContext.java

```diff
@@ -44,14 +45,18 @@
     public InsertValueContext(final Collection<ExpressionSegment> assignments, final List<Object> parameters, final int parametersOffset) {
         parameterCount = calculateParameterCount(assignments);
         valueExpressions = getValueExpressions(assignments);
-        this.parameters = getParameters(parameters, parametersOffset);
+        this.parameters = getParameters(parameters, parshardingsphere-infra-binderametersOffset);
```

Comment:
@CodingBingo Why change `parametersOffset` to this name?

Reply:
sorry, this is copy mistake.